### PR TITLE
fix(signup): add tracking error navigation

### PR DESCRIPTION
**Pull request by stif59100**
ref:DTRSD-97933

Signed-off-by: stif59100 <steeve.vanderstocken@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `release/sprint-2248-2250`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | DTRSD-97933
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

I add tracking page for error on the sign-up page.

### CODE REVIEW

**Review by mohammed-zahaf — APPROVED** [review]

**Review by sachinrameshn — DISMISSED** [review]

**Review by ghyenne — APPROVED** [review]

File: packages/manager/apps/sign-up/src/routing.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ export const state = {
     trackError: /* @ngInject */ (atInternet, me) => (step, field) => {
       atInternet.trackEvent({
         event: 'ACCOUNT_CREATION_VALIDATION',
-        page: `accountcreation-${step}-error${field}-${me.legalform}`,
+        name: `accountcreation-${step}-error${field}-${me.legalform}`,
```
Comment (ghyenne) [inline packages/manager/apps/sign-up/src/routing.js:108]: I don't see name tag for trackEvent:

atInternet.trackEvent({
page: 'pageName', // page that raised the event
event: 'eventName', // raised event
level2: '1', // (optional) if not configured in defaults you must specify your project id
visitorId: '123', // (optional) set [identifiedVisitor.id](https://identifiedvisitor.id/) sent with each hit
});

Comment (stif59100) [reply packages/manager/apps/sign-up/src/routing.js:108]: see on direct

File: packages/manager/apps/sign-up/src/routing.js [hunk 2]
```diff
@@ -103,9 +103,10 @@ export const state = {
       },
     ],
     trackError: /* @ngInject */ (atInternet, me) => (step, field) => {
-      atInternet.trackEvent({
+      atInternet.trackPage({
         event: 'ACCOUNT_CREATION_VALIDATION',
```
Comment (mohammed-zahaf) [inline packages/manager/apps/sign-up/src/routing.js:107]: you can remove this if you use `trackPage`.

File: packages/manager/apps/sign-up/src/routing.js [hunk 3]
```diff
@@ -103,9 +103,10 @@ export const state = {
...
-        page: `accountcreation-${step}-error${field}-${me.legalform}`,
+        name: `accountcreation-${step}-error${field}-${me.legalform}`,
+        type: 'navigation',
```
Comment (mohammed-zahaf) [inline packages/manager/apps/sign-up/src/routing.js:109]: to remove too.

File: packages/manager/apps/sign-up/src/routing.js [hunk 4]
```diff
@@ -103,9 +103,9 @@ export const state = {
       },
     ],
     trackError: /* @ngInject */ (atInternet, me) => (step, field) => {
-      atInternet.trackEvent({
-        event: 'ACCOUNT_CREATION_VALIDATION',
-        page: `accountcreation-${step}-error${field}-${me.legalform}`,
+      atInternet.trackPage({
+        name: `accountcreation-${step}-error${field}-${me.legalform}`,
+        customObject: { event: 'ACCOUNT_CREATION_VALIDATION' },
```
Comment (mohammed-zahaf) [inline packages/manager/apps/sign-up/src/routing.js:108]: 👍

**Review by mohammed-zahaf — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**